setup.py

```diff
@@ -75,6 +75,7 @@
     'tenacity>=8.2.3,<9',
     'catalogue>=2,<3',
     'typer<1',
+    'GitPython==3.1.43',
```

Comment:
Can you make this a range?

Reply:
I followed the default from composer https://github.com/mosaicml/composer/blob/main/setup.py#L140. Do we want to change it there too?

Reply:
Will stick with Composer's choice not to have conflicting deps